openapi.yaml

```diff
@@ -276,6 +249,66 @@ components:
         twap:
           description: The 24h time-weighted average price of the index
           type: string
+    MarketCapPercentChange:
+      type: object
+      description: The percent change over a period of time.
+      properties:
+        change24h:
+          description: Percentage change for the 24 hour period.
+          type: string
+        change7d:
+          description: Percentage change for the 7 day period.
+          type: string
+        change30d:
+          description: Percentage change for the 30 day period.
+          type: string
+    MarketCap:
+      type: object
+      properties:
+        id:
+          description: Unique identifier for the resource.
+          type: string
+          format: uuid
+        assetId:
+          description: The asset id
+          type: string
+          format: uuid
+        timestamp:
+          description: The timestamp the market capitalisation was calculated.
+          type: string
+          format: date-time
+        marketCapRank:
+          description: The market cap ranking order.
+          type: string
+        volumeRank:
+          description: The 24hr volume ranking order.
+          type: string
+        price:
+          description: The global weighted average price of the asset in USD.
+          type: string
+        volume:
+          description: The total global weighted average 24h volume of the asset.
+          type: string
+        totalSupply:
+          description: The total supply of the asset.
+          type: string
+        freeFloatSupply:
+          description: The circulating supply of the asset.
```

Comment:
Circulating supply is just another term for "free float supply"; should we explain what the free float supply is?   `The total supply less any supply that has been removed from circulation.`?

Reply:
Updated the description.